_drafts/dei-coop.md

```diff
@@ -6,17 +6,17 @@ categories:
+Tech and data science spaces tend to be overwhelmingly white. This is true here at Fred Hutch and within The Coop. I am writing this post to express The Coop's firm stance against racial injustice in our community and beyond. We are committed to uplifting and amplifying the voices of Black biologists and data scientists. We are committed to incorporating discussions of ethics into our classes on data science. We are committed to hearing from our community when we make mistakes, or when simply not enough is being done. If you have thoughts, questions, or concerns please feel empowered and encouraged to reach out to either Kate Hertwick or myself (Lauren Wolfe) through Slack, Teams, or email.
 
 ## Get involved at Fred Hutch
 
-Fred Hutch's Office of Diversity, Equity, and Inclusion (ODEI) is holding sessions to engage the Fred Hutch commuity in structured conversation about racisim.
+Fred Hutch's Office of Diversity, Equity, and Inclusion (ODEI) is holding sessions to engage the Fred Hutch commuity in structured conversation about racisim. You can register for either of the following sessions by emailing the ODEI at `diversity@fredhutch.org` and requesting a registration link.
 
-* The first session, June 3, is specifically for People of Color at the Hutch, to discuss racism’s specific impacts on People of Color and responds to the need for healing space. Register in advance [here](https://zoom.us/meeting/register/tJYlcuqgqTwsGNX0Rawg0FARCjprtTmdpIwB).
+* The first session, June 3, is specifically for People of Color at the Hutch, to discuss racism’s specific impacts on People of Color and responds to the need for healing space. 
 
-* The second session, June 10, is open to all employees. We will facilitate an honest conversation about racism and discuss what we can do to foster a greater sense of inclusion among our community — inside and outside of the Hutch. Register in advance [here](https://zoom.us/meeting/register/tJErce6prjwuHNTYjaSQWFFiTfR78rDe7xEo).
+* The second session, June 10, is open to all employees. We will facilitate an honest conversation about racism and discuss what we can do to foster a greater sense of inclusion among our community — inside and outside of the Hutch. 
 
 Year-round we have an amazing group of organizers running [Diversity Council](https://centernet.fredhutch.org/cn/u/di/erg/diversity-council.html). Diversity Council serves the Fred Hutch community by giving a voice to racial and ethnic minorities, promoting employee engagement through diversity and inclusion educational initiatives, and celebrating successes through inclusive representation in internal and external communications. Join their listserv via the link provided to learn more about upcoming events.
```

Comment:
note that this is an internal link, or look to see if there's something on the public FH site

Reply:
good catch thanks!